### ISSUE TITLE
Bug: Scheduled Tasks not working

### ISSUE THREAD

**Issue by T1000-Cyberd** [MISP/MISP#9775]
### Actual behavior

Scheduled Tasks are not working as scheduled. I am scheduling feed fetch but they are not happening anymore.

### Expected behavior

Before the last update it was working.

### Steps to reproduce

Before the last update it was working.

### Version

v2.4.193 

### Operating System

Ubuntu

### Operating System version

20.04

### PHP version

7.4

### Browser

All

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by elvidence** [comment]
Did you check your workers? Worker type: scheduler is very gentle in nature and easily gets upset ;-)

**Comment by iglocska** [comment]
You're much better off using cron jobs and console tasks (go to event actions -> automation and scroll down to get a description of how they work). You'll get way more control over the executions as well as better oversight over what is scheduled and when it is to be executed.

I would rephrase what @elvidence said so diplomatically by saying that the scheduler worker / scheduled jobs are hot garbage and I feel shame each time I am reminded of them...

**Comment by T1000-Cyberd** [comment]
@elvidence All workers are green. I rolled back one version from the backup and scheduler is working, at least on fetching feed data. @iglocska Will take a look at cron jobs, and how to schedule from there. Well hope they fix it on the next version, i also had problems fetching some feeds after the update, when i rolled back it works.
